build/target-repository/docs/rector_rules_overview.md

```diff
@@ -8699,7 +8699,7 @@ use Rector\Transform\Rector\ClassMethod\ReturnTypeWillChangeRector;
 
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->ruleWithConfiguration(ReturnTypeWillChangeRector::class, [
-        ArrayAccess::class => ['offsetGet'],
+        ArrayAccess::class,
```

Comment:
No, the methods needs to be defined, or all methods / custom methods will have `#[\ReturnTypeWillChange]`, which unexpected.

The `ReturnTypeWillChangeRector` already cover to merge new definition per needed over default config.

Reply:
I see. We were missing test for that one, that's why I assumed it's not needed.
Will complete one :+1: